QEfficient/cloud/infer.py

```diff
@@ -67,7 +67,7 @@ def main(
         ignore_patterns=["*.txt", "*.onnx", "*.ot", "*.md", "*.tflite", "*.pdf", "*.msgpack", "*.h5"],
     )
     tokenizer = AutoTokenizer.from_pretrained(
-        model_hf_path, use_cache=True, padding_side="left", trust_remote_code=True
+        model_hf_path, use_cache=True, padding_side="right", trust_remote_code=True
```

Comment:
Looks like `trust_remote_code=True` is causing issues for some models, e.g: Phi2, Phi3. I think we should remove that or make it a flag whether to use remote code or not.

Reply:
Sure, addressed this. In a larger change, Will make it part of `kwargs` as an optional flag to enable custom models, tokenizers.